webpack.config.js

```diff
@@ -16,6 +17,9 @@ var shared = {
     new webpack.DefinePlugin({
       STORYSHOP_API_URI: dev ? "'http://localhost:9999'" : "'http://api.storyshopapp.com'",
     }),
+    new CleanWebpackPlugin(['build'], {
+      exclude: ['index.html']
```

Comment:
In order to avoid a `cp` command, we removed a valid `.gitignore`, altered all the build scripts, added a cleaning script, and excluded the html from the clean. And this will require developers to try and remember not to commit the rest of the files in build. I feel things may be getting a little hacky at this point.

If the only reason for all these changes is to support Windows, I'd just favour the Windows users installing a virtual machine because it feels like we're making a lot of compromises. What do you think?